.travis.yml

```diff
@@ -93,15 +93,11 @@ script:
     - EB_BOOTSTRAP_VERSION=$(grep '^EB_BOOTSTRAP_VERSION' $TRAVIS_BUILD_DIR/easybuild/scripts/bootstrap_eb.py | sed 's/[^0-9.]//g')
     - EB_BOOTSTRAP_SHA256SUM=$(sha256sum $TRAVIS_BUILD_DIR/easybuild/scripts/bootstrap_eb.py | cut -f1 -d' ')
     - EB_BOOTSTRAP_FOUND="$EB_BOOTSTRAP_VERSION $EB_BOOTSTRAP_SHA256SUM"
-    - EB_BOOTSTRAP_EXPECTED="20170808.01 7a22faf5ee9d249807fd883e435008fb84c11129425ec09d3cc76918b36ef3d5"
+    - EB_BOOTSTRAP_EXPECTED="20180117.01 c3e828769fe30f9e4798972f7ab052632047e5df3224e002d3d2599033678745"
```

Comment:
@xdelaruelle Let's so with `20180201.01` here to match with the date that is was last updated (you'll need to change it accordingly in `easybuild/scripts/bootstrap_eb.py` too of course).